libraries/chain/controller.cpp

```diff
@@ -3979,63 +4023,47 @@ signed_block_ptr controller::fetch_block_by_id( const block_id_type& id )const {
    return signed_block_ptr();
 }
 
+bool controller::block_exists(const block_id_type&id) const {
+   auto sb_ptr = my->block_data.fork_db_fetch_block_by_id(id);
```

Comment:
What is `sb` in `sb_ptr`?

Reply:
signed_block_header. Added type to make it clear.